src/app/modules/submission/submission.service.ts

```diff
@@ -1157,6 +1157,7 @@ export const validateAttachments = (
 
 export const getQuarantinePresignedPostData = (
   attachmentSizes: AttachmentSizeMapType[],
+  growthbook: boolean | undefined,
```

Comment:
Should have a less generic name.
```suggestion
  enableGuarddutyBucketPost: boolean | undefined,
```

Reply:
Updated this! Thanks for the suggestion. Made it optional as well - I saw the function gets called in a bunch of places so don't want to touch on too many changes